CHANGELOG.md

```diff
@@ -11,12 +11,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Fix connecting user from background thread [#2762](https://github.com/GetStream/stream-chat-swift/pull/2762)
 
 ## StreamChatUI
+### ✅ Added
+- Add customization support for `ChannelListLoadingView` [#2772](https://github.com/GetStream/stream-chat-swift/pull/2772)
 ### 🐞 Fixed
 - Fix tapping on the status bar scrolling to the bottom instead of the top [#2763](https://github.com/GetStream/stream-chat-swift/pull/2763)
 - Fix empty channel header view for new DM Channels [#2764](https://github.com/GetStream/stream-chat-swift/pull/2764)
 - Fix showing copy message action when text is empty [#2765](https://github.com/GetStream/stream-chat-swift/pull/2765)
+- Fix dummy texts UI Glitch in `ChannelListLoadingView` [#2772](https://github.com/GetStream/stream-chat-swift/pull/2772)
 ### 🔄 Changed
 - Make record button in composer, visible depending on the channel's capabilities. [#2758](https://github.com/GetStream/stream-chat-swift/pull/2758)
+- Rename`Components.chatChannelListLoadingView` -> `Components.channelListLoadingView` [#2772](https://github.com/GetStream/stream-chat-swift/pull/2772)
```

Comment:
I don't think this rename is worth it. We would need to manage a deprecation, and the current name is not that bad tbh.

Reply:
The name is not consistent with the other view states: 

<img width="949" alt="image" src="https://github.com/GetStream/stream-chat-swift/assets/12814114/c43df1d5-77ed-4cb7-b808-7ffa1cde5c4b">

It does not make sense to have `channelListErrorView`, `channelListEmptyView` and then `chatChannelListLoadingView`.

This is a really simple deprecation, with one click it is fixed on the app side, since we use the "renamed" deprecation.

Reply:
We also have `isChatChannelListStatesEnabled`, and most of our types have the prefix `ChatChannelList`, so I don't think it's worth it. However, it's up you in this case.

Reply:
That is why I only changed in this place. The inconsistency is only in the `Components` config. The properties are together, so it makes sense for them to have the same pattern. 
 
Spending time reverting this now is not worth IMO, for such a simple change 🤔